lib/test/t_ComposedDistribution_std.cxx

```diff
@@ -75,7 +75,10 @@ int main(int, char *[])
     distribution.setName("myDist");
     UnsignedInteger dim = distribution.getDimension();
     fullprint << "Distribution " << distribution << std::endl;
-    std::cout << "Distribution " << distribution << std::endl;
+    std::cout << "Distribution " << std::endl;
```

Comment:
I think you should use "fullprint" here and in the following lines. It is an alias for std::cout defined at the start of the script.

Reply:
That is untrue. The variable `fullprint` is an instance of the `OStream` class. This is why the two methods `std::cout` and `fullprint` do not produce the same result.

Reply:
OK, I had not understood the subtlety.

Reply:
@josephmure It is **not** an alias for std::cout, but an output stream built upon std::cout as you can see in OStream.cxx:
```
// This operator writes Object derived objects to OStream

OStream & operator << (OStream & OS, const Object & obj)
{
  OS.getStream() << obj.__repr__();
  return OS;
}
```
and you see that it calls the __repr__() method of any object streamed to this stream.